pkg/virt-api/webhooks/validating-webhook/admitters/vms-admitter.go

```diff
@@ -670,96 +807,29 @@ func validateSnapshotStatus(ar *admissionv1.AdmissionRequest, vm *v1.VirtualMach
 	return nil
 }
 
-func (admitter *VMsAdmitter) validateVMUpdate(oldVM, newVM *v1.VirtualMachine) []metav1.StatusCause {
-	if newVM.Status.Ready {
+func (admitter *VMsAdmitter) validateVMUpdate(vm *v1.VirtualMachine) []metav1.StatusCause {
```

Comment:
Do we still need this? 
Isn't the `ValidateVirtualMachineSpec` enough?
Looks like there is already this check about the socket number and the maxGuest.
Can you explain? Thnanks